runtime/doc/support.txt

```diff
@@ -0,0 +1,55 @@
+*support.txt*			    Nvim
+
+
+                            NVIM REFERENCE MANUAL
+
+
+Supported platforms					*supported-platforms*
```

Comment:
The idea was to move this tag to the subsection below.

@justinmk what would you like as the top-level text for this file?

Reply:
Can wait on that. Just wanted to get the URL https://neovim.io/doc/user/support.html settled, so that we don't break links later :)